### DISCUSSION TITLE
Is it possible to do caching (e.g. with redis) within Vercel?

### DISCUSSION THREAD

**Discussion by OwaisSiddiqui** [vercel/vercel#5667] [Help] [answered]
I have a Next.js website and I followed this article: https://medium.com/swlh/implementing-a-simple-redis-cache-with-next-js-5103976de188 to setup a redis database for caching. According to this q-and-a on Vercel: https://vercel.com/knowledge/does-vercel-support-docker-deployments I cannot run a docker image on Vercel. I am asking would it would be possible to do caching (like in the article) within Vercel or possibly run a docker image.

**Comment by OwaisSiddiqui** [comment, marked as answer]
I just ended up using [lambda.store](https://lambda.store) (Cloud Redis storage) for caching.

**Reply by JaiMorjaria** [reply]
Is that Vercel's serverless functions? Could you link me the page for Cloud Redis storage in their documentation? Thanks
